docs/using/installing/index.rst

```diff
@@ -219,11 +229,11 @@ Using Rackspace
 Another way to get a Flocker cluster running is to use Rackspace.
 You'll probably want to setup at least two nodes.
 
-#. Create a new Cloud Server running Fedora 20
+#. Create a new Cloud Server
 
    * Visit https://mycloud.rackspace.com
    * Click "Create Server".
-   * Choose the Fedora 20 Linux distribution as your image.
+   * Choose a supported Linux distribution as your image (for example, Ubuntu 14.04 or CentOS 7).
```

Comment:
Elsewhere when we want to improve things later we use a comment with "XXX This is what to improve now FLOC-XYZ" where FLOC-XYZ is a follow up issue. It helps. Also, sometimes it is helpful for a reviewer if the compromises made are put as comments on the PR. Else I'm saying "X would be nice" and reading "we thought of X, but didn't want to spend the time" - when not spending the time is a perfectly reasonable decision to make, but it'd be nice for me not to have to do the same thinking as the implementer.